layers/cmd/valueset.go

```diff
@@ -311,6 +319,150 @@ func LoadValuesetFiles(vs *Valuesets, files []string) error {
 	return nil
 }
 
+type Options struct {
+	lookupOrder []string
+	output      []string
+	separator   map[string]string
+}
+
+func (opt Options) splitCell(header, cell string) []string {
+	sep := opt.separator[header]
+	// if separator is empty, default to space
+	if sep == "" || sep == " " {
+		return strings.Split(strings.TrimSpace(cell), " ")
+	}
+	s := strings.ReplaceAll(cell, " ", "")
+	return strings.Split(strings.TrimSpace(s), sep)
+}
+
+var options_re = regexp.MustCompile(`options`)
+
+func parseSpreadsheet(sheet [][]string) (optionsRows [][]string, headers []string, data [][]string, err error) {
+	var headerIdx int
+ROWS:
+	for rowIdx, row := range sheet {
+		if len(row) == 0 {
+			continue
+		}
+		for _, columnData := range row {
```

Comment:
If row[0] matches the re, then this is an option row. The columns[1:] are the values for the option.